lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -850,17 +850,15 @@ public function getLowerExpression($str)
     }
 
     /**
-     * Returns the SQL snippet to get the position of the first occurrence of substring $substr in string $str.
+     * Returns the SQL snippet to get the position of the first occurrence of the substring in the string.
      *
-     * @param string    $str      Literal string.
-     * @param string    $substr   Literal string to find.
-     * @param int|false $startPos Position to start at, beginning of string by default.
-     *
-     * @return string
+     * @param string      $string    String to locate the substring in, SQL expression.
```

Comment:
This comment is unclear to me - can you rephrase it, please?

Reply:
This is the best I could come up with. Otherwise, it was "The SQL expression which represents the string in which the substring should be located". And this "the SQL expression" will be in a lot of places. Is that better?

Reply:
Aha, the wording "sql expression" put after the comma did confuse me

Reply:
So what do you recommend?

Reply:
"SQL expression producing the haystack" or something like that?